src/storage/src/table/batch_table/storage_table.rs

```diff
@@ -360,6 +370,8 @@ impl<S: StateStore, SD: ValueRowSerde> StorageTableInner<S, SD> {
         pk: impl Row,
         wait_epoch: HummockReadEpoch,
     ) -> StorageResult<Option<OwnedRow>> {
+        // `get_row` doesn't support select `_rw_timestamp` yet.
+        assert!(self.epoch_idx.is_none());
```

Comment:
Can we extend the `get` interface to also return the user key (thus epoch)?

Reply:
Yes. We do have a plan to add an extended interface to return the epoch from the storage. Let's leave it for a later PR.

Reply:
See #19345

Reply:
I am going to merge this PR. Once https://github.com/risingwavelabs/risingwave/pull/19345 is merged into the main, I can switch to this new interface.